hevm.cabal

```diff
@@ -241,7 +253,8 @@ executable hevm
     vty,
     stm,
     spawn,
-    optics-core
+    optics-core,
+    eventlog2html
```

Comment:
Does this actually need to be in the cabal file? Can it go in the nix shell instead? I would prefer that it's not listed as a direct dep of the exe unless actually required...

Reply:
Yeah, good point. Moved to nix :)